### PR TITLE
Update CHANGELOG for v1.3.0 release

### PR DESCRIPTION
**Pull request by azdagron**

### CODE REVIEW

**Review by rturner3 — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+
+### Added
+- Experimental Windows support (https://github.com/spiffe/spire/projects/12)
+- Ability to revert SPIFFE cert validation to standard X.509 validation in Envoy (#3009,#3014,#3020,#3034)
+- Configurable leader election resource lock type for the K8s Workload Registrar (#3030)
+- Ability to fetch JWT SVIDs and JWT Bundles on behalf of workloads via the Delegated Identity API (#2789)
+- CanReattest flag to NodeAttestor responses to facilitate future features (#2646)
+
+### Fixed
+- Spurious message to STDOUT when a plugin no plugin_data section configured (#2927)
```
Comment (amartinezfayo) [inline CHANGELOG.md:13]: Does this read weird?
```suggestion
- Spurious message to STDOUT when no plugin_data section is configured in a plugin (#2927)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+
+### Changed
+- SPIRE entries with malformed parent or SPIFFE IDs are removed on server startup (#2965)
+- SPIRE no longer prepends slashes to paths passed to the API when missing (#2963)
+- K8s Workload Registrar retries up 5 seconds to connect to SPIRE Server (#2921)
```
Comment (amartinezfayo) [inline CHANGELOG.md:18]:
```suggestion
- K8s Workload Registrar retries up to 5 seconds to connect to SPIRE Server (#2921)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+- Improved error messaging when unauthorized resources are requested via SDS (#2916)
+- Small documentation improvements (#2934,#2947,#3013)
```
Comment (amartinezfayo) [inline CHANGELOG.md:20]: nit:
```suggestion
- Small documentation improvements (#2934, #2947, #3013)
```

**Review by amartinezfayo — COMMENTED** [review]
Looks great! I have some minor comments.

**Review by amartinezfayo — APPROVED** [review]

**Pull request merged**